### PR TITLE
Smaller icons for smaller headlines in emails even in Outlook

### PR DESCRIPTION
**Pull request by lmath**
## What does this change?
In emails, when headlines have icons (e.g. the quote marks for opinion headlines), then the size of the icon should be consistent with the size of the headline. This is the case for most clients. This change should ensure this is also true for Outlook on Windows. 

This involved setting height on icon img elements. This will be ignored by most clients but will be useful for Outlook.

## What is the value of this and can you measure success?
Emails read on Outlook on Windows will be rendered more consistently with how they appear on other clients. 

## Does this affect other platforms - Amp, Apps, etc?
Nope.

## Screenshots
This is how the icons appear on other clients e.g. here iPhone 6:
image

This is how this email would look on Outlook on Windows before the change:
image

After the change:
image

### CODE REVIEW

File: facia/app/views/fragments/emailFrontBody.scala.html [hunk 1]
```diff
@@ -46,9 +46,9 @@ <h4 class="trail-text">@Html(trailText)</h4>
     }
 }
 
-@headlineAndTrailWithCutout(card: ContentCard) = {
+@headlineAndTrailWithCutout(card: ContentCard, withImage: Boolean) = {
     @fullRow(Seq("facia-card__text")) {
-        @headline(card)
+        @headline(card, withImage)
```
Comment (joelochlann) [inline facia/app/views/fragments/emailFrontBody.scala.html]: Could be slightly clearer to use a named param here, i.e. `@headline(card, largeHeadline = withImage)`

File: facia/app/views/fragments/emailFrontBody.scala.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
 @import views.support.TrailCssClasses.toneClassFromStyle
 
 
-@headline(card: ContentCard) = {
+@headline(card: ContentCard, largeHeadline: Boolean = false) = {
```
Comment (joelochlann) [inline facia/app/views/fragments/emailFrontBody.scala.html]: maybe `isLarge` would be a better name for this param? since we already know it's a headline

**Review by joelochlann — APPROVED** [review]
Good stuff! Thanks for fixing. Just a few naming things

File: facia/app/views/fragments/emailFrontBody.scala.html (hunk 2 above)
Comment (lmath) [inline facia/app/views/fragments/emailFrontBody.scala.html]: Agreed. That is better.

**Pull request merged**